mongo-java-driver/build.gradle

```diff
@@ -40,6 +29,7 @@ dependencies {
     compile "io.netty:netty-buffer:$nettyVersion", optional
     compile "io.netty:netty-transport:$nettyVersion", optional
     compile "io.netty:netty-handler:$nettyVersion", optional
+    compile project(path: ':driver')
```

Comment:
This is necessary to pull in the shaded classmate packages?  How does it interact with the srcDirs on line 49?